config.json

```diff
@@ -1,6 +1,7 @@
 {
   "language": "Go",
   "active": true,
+  "blurb": "Go is a compiled, open source programming language with a small, consistent syntax, a powerful standard library and fantastic tooling. It's a great fit for web backends and command-line tools.",
```

Comment:
The current blurb is already at the word limit. The 'about' is going to show farther down the page, so as long as we're happy with how the about doc expands on the blurb, I think we should be good with the shorter one.